webapp/src/Entity/Problem.php

```diff
@@ -30,7 +30,8 @@
 #[ORM\Index(columns: ['special_compare'], name: 'special_compare')]
 #[ORM\HasLifecycleCallbacks]
 #[UniqueEntity(fields: 'externalid')]
-class Problem extends BaseApiEntity
+class Problem extends BaseApiEntity implements HasExternalIdInterface,
```

Comment:
```suggestion
class Problem extends BaseApiEntity implements
    HasExternalIdInterface,
```

Reply:
Is this in PSR? I'm not sure if I like it more

Reply:
PHPCS complains so I assume so...

The reason I           like it is that now all implemented things are neatly on the same vertical line,
The reason I don't like it is that this reads very unnatural.

```
Lists of implements and, in the case of interfaces, extends MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one interface per line.
```

https://www.php-fig.org/psr/psr-12/ ctrl+f `implements`